staff/roberto-perez/mailchimp-template/style.sass

```diff
@@ -0,0 +1,260 @@
+$backGroundColor: #45BCD2
+
+* 
+    box-sizing: border-box
+
+html,
+body 
+    margin: 0
+    padding: 0
+    width: 100%
+    height: 100%
+
+html 
+    font-size: 1em
+    line-height: 1.15
+
+body 
+    line-height: 1.35
+    
+body 
+    font-family: 'Roboto', sans-serif
+    font-size: 16px
+    font-weight: normal
+    background-color: $backGroundColor
+
+.l-wrapper
+    min-height: 100%
+    display: grid
+    grid-template-rows: auto 1fr auto
+    grid-template-columns: 100%
+
+.l-content
+    max-width: 1200px
+    padding: 0px 15px
+    margin-left: auto
+    margin-right: auto
+    width: 990px
+
+.b-button
+    border-radius: 7px
+    text-align: center
+    text-decoration: none
+    font-size: 14px
+    color: #fff
+    &:hover
+        -webkit-box-shadow: inset 0px 0px 5px 0px rgba(0,0,0,0.75)
+        -moz-box-shadow: inset 0px 0px 5px 0px rgba(0,0,0,0.75)
+        box-shadow: inset 0px 0px 5px 0px rgba(0,0,0,0.75)
+    &--red 
```

Comment:
did you use a (linear-gradient) generator ?

Reply:
Yeah! Do you ask for the prefix -moz- and -webkit- :laughing:? I'm not sure if they are necessary for modern browsers